### PR TITLE
Refreshing project readme

### PR DESCRIPTION
**Pull request by kcoleman731**
*Description of changes:*

This pull request updates a decent amount of content in our Readme. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,21 +113,27 @@ Deploy the stack using the following command
 cdk deploy
 ```
 
-### Examples 
+This will provision the following:
 
-To view more examples for how you can leverage `cdk-eks-blueprint`, see the `examples` directory.
+- [x] A new Well-Architected VPC with both Public and Private subnets.
```
Comment (shapirov103) [inline README.md:118]: Do we need a WA review before we call this out? I am concerned that if the cluster does not align with any of the WA recommendation for EKS, then a partner trying to get certification will point to this section.

File: README.md [hunk 2]
```diff
@@ -110,21 +113,27 @@ Deploy the stack using the following command
...
+- [x] AWS and Kubernetes resources needed to support [AWS Load Balancer Controller](https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html).
 
 ---
 
-## Why should I use this framework?  
+## Why should I use this QuickStart?  
 
-SSPs can be difficult to design and build. From an operations perspective, you need to determine the right mix of tools and services you want to include in your platform and how they integrate. You also need to determine how to maintain a fleet of clusters and their associated addons across regions. From a developer perspective, you need to determine how to onboard an operate workloads on the platfrom as well. 
+The ecosystem of tools that have developed around Kubernetes and the Cloud Native Computing Foundation (CNCF) provide cloud engineers with a wealth of choice when it comes to architecting their infrastructure. Determining the right mix of tools and services however, in addition to how they integrate, can be a challenge. As your Kubernetes estate grows, managing configuration for your clusters can also become a challenge.
```
Comment (shapirov103) [inline README.md:130]: provide = provides?

File: README.md [hunk 3]
```diff
@@ -110,21 +113,27 @@ Deploy the stack using the following command
...
 
-The `cdk-eks-blueprint` framework provides logical abstractions and perscriptive guidance for building a well-architected platform, and ultimately, will help accelerate time to market for your platform implementation.
+AWS customers are building internal platforms to tame this complexity, automate the management of their Kubernetes environments, and make it easy for developers to onboard their workloads. These platforms require investment of time and engineering resources to build however. The goal of this QuickStart is to provide customers with a tool chain that can help them deploy a Well-Architected platform on top of EKS with ease. The `cdk-eks-blueprint` framework provides logical abstractions and prescriptive guidance for building a platform. Ultimately, we want to help EKS customers accelerate time to market for their own platform initiatives.
```
Comment (shapirov103) [inline README.md:132]: minor: "to build however"? May use commas and move "however" to the beginning.

**Review by shapirov103 — COMMENTED** [review]
minor comments

File: README.md [hunk 4]
```diff
@@ -1,40 +1,45 @@
...
-To view a library of sample implementations, please see our [SSP Patterns Repository](https://github.com/shapirov103/eks-ssp-patterns).
+To view a library of examples for how you can leverage the `cdk-eks-blueprint`, please see our [SSP Patterns Repository](https://github.com/shapirov103/eks-ssp-patterns).
 
-You can also view a sample implementation in `bin/main.ts`.
+You can also find a sample implementation that resides in this repository in `bin/main.ts`.
 
 ## Getting Started
 
-Install CDK matching the version of the SSP Quickstart.
+First, make sure you have the [`aws-cli`](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed. To verify your installation, run the following:
```
[the remark this answers is not in the document]
Comment (kcoleman731) [reply README.md:35]: Agree.

File: README.md (hunk 1 above)
Comment (kcoleman731) [reply README.md:118]: Fine by me

File: README.md (hunk 2 above)
Comment (kcoleman731) [reply README.md:130]: yep

File: README.md (hunk 3 above)
Comment (kcoleman731) [reply README.md:132]: fine by me

File: README.md (hunk 1 above)
Comment (shapirov103) [reply README.md:118]: We should, I am just pointing that it is a distinct activity that requires drilling into the WA spec. I know, WA is slow to deliver container lense. I am just curious about what specific guidelines are WA for EKS before we can call out we are compliant.

**Review by shapirov103 — APPROVED** [review]

**Pull request merged**